packages/mdc-notched-outline/mdc-notched-outline.scss

```diff
@@ -53,11 +55,9 @@
 .mdc-notched-outline__path {
```

Comment:
Not sure if you want to change it here or another PR, but line 50 needs to be updated: change `opacity 100ms ease` to use the new duration variable and standard curve timing.
Also include a transition for `border-color` for the transition on hover.

Reply:
In the `.mdc-notched-outline__idle` selector (GitHub won't let me comment there)

Reply:
I updated the branch with all of the suggested changes. Given the brevity of the overall duration, this seems like it might actually be fine (and even if I slow it down in dev tools it still doesn't seem like the half-faded line and the text don't end up intersecting too badly).

As much as I'd like to avoid transitioning on non-compositable properties, adding border-color fixes another issue I'd noticed, so thanks for pointing that out.